Lesson2/WorkNotes.py

```diff
@@ -0,0 +1,51 @@
+poem_text = """I wandered lonely as a cloud
```

Comment:
Still not all linter errors were fixed:
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:1:1: D100 Missing docstring in public module
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:6:38: W291 trailing whitespace
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:16:7: W291 trailing whitespace
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:23:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:25:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:27:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:27:80: E501 line too long (97 > 79 characters)
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:28:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:33:7: W291 trailing whitespace
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:40:1: D103 Missing docstring in public function
./T-HOMETASKS/L02/l02_h01_PukichYurii.py:50:27: W292 no newline at end of file

Key points:
Docstring with task description  - this will simplify your and my life )
Long strings >79 characters ( better to cut them for example by reducing variables names
CTRL+ALT+L shortcut in pycharm should solve other issues ( spaces and new line at the end)